### ISSUE TITLE
Exporting to credential file throws error

### ISSUE THREAD

**Issue by thomscode** [benkehoe/aws-export-credentials#17]
I have the following `~/.aws/config` file on an EC2. I know it seems redundant as it isn't assuming a role, but I need the named profile for my use case:

``` 
[profile myprofile]
credential_source = Ec2InstanceMetadata
region = us-west-2
output = json
```
When running `aws-export-credentials --credentials-file-profile myprofile` I receive the following error:

```
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/aws-export-credentials", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/.local/lib/python3.10/site-packages/aws_export_credentials/aws_export_credentials.py", line 269, in main
    write_values(session, args.credentials_file_profile, values)
  File "/home/ubuntu/.local/lib/python3.10/site-packages/aws_export_credentials/config_file_writer.py", line 54, in write_values
    session.get_config_variable('credentials_file'))
AttributeError: 'Session' object has no attribute 'get_config_variable'
```

**Comment by benkehoe** [comment]
It looks like I missed something when converting it from botocore to boto3. I've shipped v0.18 with a fix.

**Comment by thomscode** [comment]
Confirmed, that solved it! Thanks for the quick fix.
